class.jetpack.php

```diff
@@ -7114,7 +7114,16 @@ public static function is_akismet_active() {
 			if ( ! $akismet_key ) {
 				return false;
 			}
-			$akismet_key_state = Akismet::verify_key( $akismet_key );
+			$cached_key_verification = get_transient( 'jetpack_akismet_key_is_valid' );
+
+			// We cache the result of the Akismet key verification for ten minutes.
+			if ( in_array( $cached_key_verification, array( 'valid', 'invalid', 'failed' ) ) ) {
+				$akismet_key_state = $cached_key_verification;
+			} else {
+				$akismet_key_state = Akismet::verify_key( $akismet_key );
+				set_transient( 'jetpack_akismet_key_is_valid', $akismet_key_state, 10 * MINUTE_IN_SECONDS );
```

Comment:
The only concern would be caching a `failed` response, since it isn't indicative of the key status itself, but between the given issue of avoiding an plethora of front-end external requests and the cache only lasting 10 minutes, I'm good with it